idl/mutation.idl.hh

```diff
@@ -168,3 +175,7 @@ class mutation_fragment stub [[writable]] {
                    partition_start, partition_end> fragment;
 };
 
+class mutation_fragment_v2 stub [[writable]] {
+    std::variant<clustering_row, static_row, range_tombstone_change,
+                   partition_start, partition_end> fragment;
+};
```

Comment:
We should start using it in rpc.

Reply:
But didn't we have something similar when we changed the half-reversed reads?

Reply:
ref: 8c257db28322a134e131ae2200200c6f878a5c8a

Reply:
This has no connection with reversed reads, those use the query-result to transmit results to the coordinator.
The only user of `frozen_mutation_fragment` (v1) is repair and streaming and indeed it is high time to switch to v2, so we can start the deprecation clock for v1 mutation fragment, which is left alive just for these two users.

Reply:
Right. @asias